clientgui/sg_TaskPanel.cpp

```diff
@@ -29,13 +29,8 @@
 
 
 #define SORTTASKLIST 1  /* TRUE to sort task selection control alphabetically */
-#define SLIDESHOWWIDTH ADJUSTFORXDPI(290)
```

Comment:
I feel it is better to leave these #define statements, just eliminating ADJUSTFORXDPI() and ADJUSTFORYDPI(), and restore them where they were used before instead of using a number. This is a form of comment in the places they are used, to help others understand how the values were obtained. An example is the old line 514:
` m_SlideShowArea->SetMinSize(wxSize(SLIDESHOWWIDTH+(2*SLIDESHOWBORDER), SLIDESHOWHEIGHT+(2*SLIDESHOWBORDER)));`
The compiler will replace these with the computed values, so the run time efficiency is not reduced by doing this.